<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>490</y>
      <width>50</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>490</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>150</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>4</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>261</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exit">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>120</y>
       <width>111</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>K</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>190</y>
       <width>111</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>S</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ok">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>260</y>
       <width>111</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>550</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <item>
     <property name="text">
      <string>user1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>user2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>user3</string>
     </property>
    </item>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_withdrawal">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>50</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Cash Withdrawal</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_deposit">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>120</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Deposit Money</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_transfer">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>190</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Transfer Money</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_nameSurname">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_balance">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_iban">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>261</y>
       <width>152</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exit_2">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>190</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>EXIT</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_20">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>120</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>20</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_50">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>190</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>50</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_100">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>50</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_200">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>120</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>200</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>190</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>25</pointsize>
      </font>
     </property>
     <property name="maxLength">
      <number>7</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_cancel_2">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>260</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>CANCEL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ok_2">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>260</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Please select the 
 amount you wish to
withdraw or enter it
using the keypad</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_balance_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>120</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_iban_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>190</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>521</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_cancel_3">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>260</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>CANCEL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_ok_3">
     <property name="geometry">
      <rect>
       <x>361</x>
       <y>260</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>160</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>Please enter the IBAN
 you wish to send
money to.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>122</x>
       <y>140</y>
       <width>271</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>143</y>
       <width>41</width>
       <height>43</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
